README.md

```diff
@@ -441,8 +444,7 @@ Fields listed under `task_definition` correspond to fields that will be included
 
 * `task_role_arn` should be the ARN of an IAM role. **NOTE**: If this role does not have the proper permissions/trust relationships on it, the `up` command will fail.
 
-* `services` correspond to the services listed in your docker compose file, with `service_name` matching the name of the container you wish to run. Its fields will be merged into an [ECS Container Definition](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html).
-The only field you can specify on it is `essential`. The default value for the essential field is true.
+* `services` correspond to the services listed in your docker compose file, with `service_name` matching the name of the container you wish to run. Its fields will be merged into an [ECS Container Definition](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html). If the `essential` field is not specified, the value defaults to true. If you are using Docker compose version 3, the `cpu_shares`, `mem_limit`, and `mem_reservation` fields are optional and must be specified in the ECS params file rather than the compose file. In Docker compose version 2, the `cpu_shares`, `mem_limit`, and `mem_reservation` fields can be specified in either the compose or ECS params file. If they are specified in the ECS params file, the values will override values present in the compose file.
```

Comment:
Could we perhaps separate each of the subfields into its own bullet point (like the network config section)? 
I realize this isn't really part of your add, but maybe it might help to link to corresponding AWS docs for `essential`.
For the new fields, what do you think about including what fields in an ECS task definition these correspond to?